database/model/public/facilities.md

```diff
@@ -44,3 +45,7 @@ Street address of the facility.
 {% docs facilities__catchment_id %}
 Catchment area ([Reference Data](#!/source/source.tamanu.tamanu.reference_data)).
 {% enddocs %}
+
+{% docs facilities__is_sensitive %}
+Controls the sync behaviour of this facility. If **`true`** encounters here will only sync to instances of this facility and nowhere else.
```

Comment:
What do we mean here by "encounters here will only sync to instances of this facility"? Are we just not syncing up encounters created on this facility?